Grbl_Esp32/grbl_trinamic.cpp

```diff
@@ -138,15 +192,25 @@ void Trinamic_Init() {
     machine_trinamic_setup();
     return;
 #endif
-#ifdef X_TRINAMIC
+#ifdef X_TRINAMIC    
     TRINAMIC_X.begin(); // Initiate pins and registries
-    trinamic_test_response(TRINAMIC_X.test_connection(), "X");
+    trinamic_test_response(TRINAMIC_X.test_connection(), "X");    
     TRINAMIC_X.toff(TRINAMIC_DEFAULT_TOFF);
     TRINAMIC_X.microsteps(settings.microsteps[X_AXIS]);
     TRINAMIC_X.rms_current(settings.current[X_AXIS] * 1000.0, settings.hold_current[X_AXIS] / 100.0);
     TRINAMIC_X.en_pwm_mode(1);      // Enable extremely quiet stepping
     TRINAMIC_X.pwm_autoscale(1);
 #endif
+#ifdef X2_TRINAMIC    
```

Comment:
This code block is just crying to be a subroutine with args like (&TRIAMINIC_X2, X_AXIS, "X2").  Since it's an init, there should be no performance considerations.